test/presubmit-tests-with-pipeline-deployment.sh

```diff
@@ -66,10 +66,24 @@ DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" > /dev/null && pwd)"
 
 echo "presubmit test starts"
 source "${DIR}/test-prep.sh"
+
+# Deploy Kubeflow
 source "${DIR}/deploy-kubeflow.sh"
 
-# Install Argo
-source "${DIR}/install-argo.sh"
+# Install Argo CLI
```

Comment:
this seems duplicate with install-argo.sh. is install-argo.sh still needed? if not you could modify that file to not install argo in the cluster. 

would be nice to keep this file concise.

Reply:
Done.